crafters/image/ImageResizer/tests/test_imageresizer.py

```diff
@@ -23,3 +33,41 @@ def test_resize():
     img_array = create_random_img_array(img_width, img_height)
     crafted_doc = crafter.craft(img_array)
     assert crafted_doc['blob'].shape[:-1] == output_dim
+
+
+def test_resize_gray():
+    img_width = 20
+    img_height = 17
+
+    # Test for int target_size
+    output_dim = 71
+    crafter = ImageResizer(target_size=output_dim)
+    img_array = create_random_gray_img_array(img_height, img_width)
+    crafted_doc = crafter.craft(img_array)
+    assert min(crafted_doc['blob'].shape[:-1]) == output_dim
+
+    # Test for tuple/list target_size
+    output_dim = (img_height, img_width)
+    crafter = ImageResizer(target_size=output_dim)
+    img_array = create_random_gray_img_array(img_width, img_height)
+    crafted_doc = crafter.craft(img_array)
+    assert crafted_doc['blob'].shape == output_dim
+
+
+def test_resize_gray_2d():
```

Comment:
can these 2 tests unified parametrizing the input image?

Reply:
the asserts are different. I am not sure if it is possible.

Reply:
how are they different the asserts of `test_resize_gray_2d` and `test_resize_gray`?  I just see the difference in the method used to create the random arrays?